src/components/Todo/types.d.ts

```diff
@@ -0,0 +1,16 @@
+import {StackNavigationProp} from '@react-navigation/stack';
+import {TodoStackParamsList} from './TodoNavigator';
+import {ModalStackParamsList} from '../../../App';
```

Comment:
can we use `src/App` instead of `../../../App`?

Reply:
Used this before adding package to `src` folder, will remove.